src/qml/RelationCombobox.qml

```diff
@@ -32,7 +32,7 @@ Item {
     ComboBox {
       id: comboBox
 
-      property var currentValue: value
+      property var myCurrentValue: value
```

Comment:
currentValue has been introduced as a readOnly property of ComboBox in Qt 5.14

Reply:
I don't like the "my" - maybe `comboBoxCurrentValue`